test/classes/initializers/compilerGenerated/syncFieldDefaultInit.chpl

```diff
@@ -1,5 +1,12 @@
 class Foo {
   var x$: sync int = 5;
+
+  // TODO: compiler should generate this (formal without sync)
+  /*
+  proc init(x: int = 5) {
+    this.x$ = x;
+  }
+  */
```

Comment:
I think this arguably could be removed (and I think I should file an issue capturing the feature request).

Reply:
Alright, I've removed it. The issue would be a good one to link to from the PR description's future work section.